DesignSafe-CI/portal
# Task/des 2306 Allow Metric Data only at DOI level

## Overview: ##
DesignSafe Project and Publications should report reuse metrics but only at the DOI level. It looks like Field Research -Social Science Collection projects have them (Data Reuse "View Collection Metrics"). Since this is a level below Mission, this is wrong. 
Simulation, Experimental, and Hybrid Simulation type projects appear to report properly.
FR Mission: Research Planning Collection, FR Mission: Engineering/Geoscience Collection, and FR Documents appear to report correctly. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2306](https://tacc-main.atlassian.net/browse/DES-2306)

## Summary of Changes: ##
Removed 'Data Reuse' from FR Amends (Document & Mission)
Removed 'Data Reuse' from FR Published Mission - Social Science Collections section

## Testing Steps: ##
1. In my Projects, find an FR. Go through the Amends pipeline and make sure 'Data Reuse' does not appear. 
2. Also in my Projects, confirm that 'Data Reuse' does not exist in any project types.
3. in Published, find an FR project. Verify that 'Download/View/Citation' Details link appears and works for Documents and Missions. Also check that the metric data does not appear in any of the sub levels to Missions. 
4. While in Published, check the other project types and verify that the 'Download/View/Citation' Details link appears and works at the correct level. Check that metric data does not appear elsewhere.

## UI Photos:
![Screen Shot 2024-01-11 at 3 10 42 PM](https://github.com/DesignSafe-CI/portal/assets/35277477/04a1287a-dbe9-4cf2-8c12-4fe2d3c50842)

## Notes: ##
